cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/LoadCSVTest.scala

```diff
@@ -147,6 +148,8 @@ can be used to instruct Neo4j to perform a commit after a number of rows.
 This reduces the memory overhead of the transaction state.
 By default, the commit will happen every 1000 rows.
 For more information, see <<query-using-periodic-commit-hint>>.
+
+Note: The <<query-use, `USE` clause>> cannot be used together with the `PERIODIC COMMIT` clause.
```

Comment:
```
<<query-use, `USE` clause>>
```
Having the backticks on `USE` makes it look like the link is just the word `clause`. Would it be better to not have backticks on `USE` so the whole thing looks like a link or keep it as it is?

Reply:
as above